pom.xml

```diff
@@ -113,6 +113,21 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>uk.ac.ebi.eva</groupId>
+            <artifactId>variation-commons-mongodb</artifactId>
+            <version>0.7-SNAPSHOT</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/src/test/resources/variation-commons-mongodb-0.7-SNAPSHOT.jar</systemPath>
```

Comment:
System path with JAR is a bad idea. As I had suggested in an email conversation on Mar 20, 2019, please use the top-level POM for **variation-commons** and bring in the dependencies.

Reply:
Hi Sundar, you needn't look at this PR 149 as it fails Travis checks and so I started fresh with PR 150.

Reply:
Sorry about that! I thought this was another PR...

Reply:
My bad, let me just close this pull request to avoid confusion. :D